app/blog/2018-01-25-polymer-2-4.md

```diff
@@ -0,0 +1,90 @@
+---
+title:  "Polymer 2.4: Paving the way for 3.0 and TypeScript support"
+---
+
+Today we released Polymer 2.4. This is an incremental release that adds a few new features for use today, and others in preparation for the upcoming 3.0 release.  Many of these changes were already released to the `3.0-pre.6` preview on npm last week, and today's release makes these changes available as an official release via Bower for 2.x users.  (As a reminder, the ES module-based Polymer 3.0 source on npm is 100% generated from the Polymer 2.x HTML source using the [modulizer](https://github.com/Polymer/polymer-modulizer) migration tool, so all changes to 3.0 are being made either via changes to the 2.x source or the modulizer conversion tool itself).
```

Comment:
Instead of the parenthetical, how about we set "As a reminder..." off in its own paragraph.